src/components/TripCard/container.js

```diff
@@ -0,0 +1,48 @@
+import React, { PureComponent } from 'react';
+import XDate from 'xdate';
+import { TripCardUI } from './ui';
+import { TripType } from '../../constants';
+
+export class TripCardContainer extends PureComponent {
+    static propTypes = {
+        trip: TripType
+    };
+
+    static defaultProps = {
+        trip: null
+    };
+
+    formatBookerName() {
+        const { trip } = this.props;
+        return `${trip.bookedBy.firstName} ${trip.bookedBy.lastName}`;
```

Comment:
if this trip was null (as it is in the default proptypes), i think this would crash. should we make it more defensive by using lodash get?